install/0000_80_machine-config_03_rbac.yaml

```diff
@@ -71,3 +71,89 @@ subjects:
 - kind: ServiceAccount
   name: prometheus-k8s
   namespace: openshift-monitoring
+---
+# Role host-networking-services lets system:node read config maps. This is needed in order to allow
+# configuring log level (and in the future more parameters) of static pods deployed in the
+# openshift-*-infra namespace.
+# Because host networking components right now only run on on-prem platforms, we create RoleBinding
+# explicitly only in namespaces for OpenStack, BareMetal, vSphere and Nutanix.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: host-networking-services
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
why do we need to include all these new manifests in ibm-cloud-managed: "true"?

Reply:
We don't, it was copy-pasta. I will remove it

Reply:
sorry I meant ibm-cloud-managed: "true" is hypeshift payload. So the question was do we want this to apply in hypershift land?

Reply:
No, we definitely don't need&want it in Hypershift. It's related to on-prem components that will not exist in Hypershift world in any foreseeable future.

Anyway, even after removing this annotation the failure signature seems to be like before... Meaning, somehow Hypershift still consumes this? More debugging will be needed...